.docker/Dockerfile.distroless

```
@@ -0,0 +1,9 @@
+FROM gcr.io/distroless/cc
+
+
+COPY --chown=4001:4001 ./polymesh /
```

Shouldn't this line be:
```suggestion
COPY --chown=4001:4001 ./polymesh /usr/local/bin/polymesh
```

---

I wasn't building a distroless image, but have just added this and will test now.

---

It will build, but trying to run it should fail, since the entrypoint is set to `/usr/local/bin/polymesh`, which will not exist.  The polymesh binary is being copied to the root as `/polymesh`.